changelog/4.4.1_2024-10-28/4502

```diff
@@ -0,0 +1,6 @@
+Bugfix: File size becomes 0 after a local updated
+
+The local copy of a file hasn't been removed after a local update, so the file size has been fixed.
```

Comment:
i'd write it a little different, since the perfect tense makes it like a past fact, when the non-removal is present (the fix you did): `The local copy of a file is not removed after a local update anymore. Therefore, the file size has been fixed.`

(just a proposal)

Reply:
I agree with you! Changed 👍🏻